env.example.json

```diff
@@ -80,5 +80,6 @@
   "-------- debug options --------": 0,
   "DEBUG_CORE": false,
   "DEBUG_PLUGINS": false,
-  "DEBUG_VERBOSE_ERRORS": false
+  "DEBUG_VERBOSE_ERRORS": false,
+  "DISABLE_WARNINGS": false
```

Comment:
Since we are using cleaner-config now, you need add this to the config file: https://github.com/EdgeApp/edge-react-gui/blob/develop/config.js

Reply:
Oh. Fixed it then.